src/core/sys/posix/arpa/inet.d

```diff
@@ -158,6 +158,31 @@ else version (NetBSD)
     const(char)*    inet_ntop(int, in void*, char*, socklen_t);
     int             inet_pton(int, in char*, void*);
 }
+else version (OpenBSD)
+{
+    alias uint16_t in_port_t;
+    alias uint32_t in_addr_t;
```

Comment:
Please use the assignment style syntax, i.e. `alias in_port_t = uint16_t`.

Reply:
again, do you want me to correct the other version sections as well? (i.e.: lines 63, 64, 88, 89, 113, 114, etc.)?

Reply:
See above.

Reply:
fixed.